### DISCUSSION TITLE
Is there a development Docker image?

### DISCUSSION THREAD

**Discussion by ITconsultants247** [cortezaproject/corteza-server#1852] [General]
I primarily work on Windows OS with experience in C#, .NET, and MS SQL. I am not familiar with Linux or Go, but I see great potential in this project and want to learn everything about its source code and how it works.

My question is: Is there a Docker image available for the source code with developer instructions on running the server side, client side, or both together? This would allow me to explore the code and understand how it works. Additionally, I've found that the documentation in the Platform Developer Guide is incomplete and not up to date. A Docker image of the full source code would help close the gaps in the developer guide and save developers like me time on setting up the development environment.

**Comment by Fajfa** [comment]
No such docker image currently. If someone has the time they can get started on it. Otherwise we'll prioritize it when time comes.
For now you'll have to clone the repo to explore it locally

**Reply by ITconsultants247** [reply]
Thank you for your reply. I truly appreciate your team for creating this wonderful product.

I have been trying for days to clone the repository and set up a development environment without success. Due to my lack of experience with the technologies used, I keep encountering issues related to Linux or third-party dependencies that are not directly related to the Corteza project.

In my humble opinion, having a Docker image for development should be a high priority. It would open this project to a variety of developers with different backgrounds, allowing the contributor base to grow and the product to advance in all areas, including documentation.
